apps/base-docs/sidebar.ts

```diff
@@ -610,6 +610,15 @@ export const sidebar: Sidebar = [
       },
     ],
   },
+  {
```

Comment:
this placement puts WalletApp with a single link above Identity that has a whole bunch of stuff. i think i'd recommend this for now:
<img width="279" alt="image" src="https://github.com/user-attachments/assets/17cef4db-c146-4302-8ef9-392acceb65f4" />